src/Kernels/Time.h

```diff
@@ -90,6 +91,9 @@ namespace seissol {
 }
 
 class seissol::kernels::Time : public TimeBase {
+  protected:
+  seissol::initializer::parameters::DamagedElasticParameters damagedElasticParameters;
```

Comment:
Do you really want to store a copy here?

Reply:
Modified to a pointer